sqlmesh/core/console.py

```diff
@@ -2643,3 +2736,73 @@ def _format_audits_errors(error: NodeAuditsErrors) -> str:
         msg = msg.replace("\n", "\n  ")
         error_messages.append(msg)
     return "  " + "\n".join(error_messages)
+
+
+def _format_evaluation_model_interval(snapshot: Snapshot, interval: Interval) -> str:
+    if snapshot.is_model and (
+        snapshot.model.kind.is_incremental
+        or snapshot.model.kind.is_managed
+        or snapshot.model.kind.is_custom
+    ):
+        # include time if interval < 1 day
+        if (interval[1] - interval[0]) < datetime.timedelta(days=1).total_seconds() * 1000:
+            return f"insert {to_ds(interval[0])} {to_datetime(interval[0]).strftime('%H:%M:%S')}-{to_datetime(interval[1]).strftime('%H:%M:%S')}"
+        return f"insert {to_ds(interval[0])} - {to_ds(interval[1])}"
+    return ""
+
+
+def _justify_evaluation_model_info(
+    text: str,
+    length: int,
+    justify_direction: t.Literal["left", "right"] = "left",
+    dots_side: t.Literal["left", "right"] = "left",
+    prefix: str = "",
+    suffix: str = "",
+) -> str:
+    """Format a model evaluation info string by justifying and truncating if needed.
+
+    Args:
+        text: The string to format
+        length: The desired number of characters in the returned string
+        justify_direction: The justification direction ("left" or "l" or "right" or "r")
+        dots_side: The side of the dots if truncation is needed ("left" or "l" or "right" or "r")
+        prefix: A prefix to add to the returned string
+        suffix: A suffix to add to the returned string
+
+    Returns:
+        The justified string, truncated with "..." if needed
+    """
+    full_text = f"{prefix}{text}{suffix}"
+    if len(full_text) <= length:
+        return (
+            full_text.ljust(length)
+            if justify_direction.startswith("l")
+            else full_text.rjust(length)
+        )
+
+    trunc_length = length - len(prefix) - len(suffix)
+    truncated_text = (
+        "..." + text[-(trunc_length - 3) :]
+        if dots_side.startswith("l")
+        else text[: (trunc_length - 3)] + "..."
+    )
+    return f"{prefix}{truncated_text}{suffix}"
+
+
+def _create_evaluation_model_annotation(snapshot: Snapshot, interval_info: t.Optional[str]) -> str:
+    if snapshot.is_audit:
+        return "run standalone audit"
+    if snapshot.is_model and snapshot.model.kind.is_external:
+        return "run external model audits"
+    if snapshot.model.kind.is_seed:
+        return "insert from seed file"
+    if snapshot.model.kind.is_full:
+        return "full refresh"
+    if snapshot.model.kind.is_view:
+        return "recreate view"
+    if snapshot.model.kind.is_incremental_by_unique_key:
+        return "insert or update rows"
+    if snapshot.model.kind.is_incremental_by_partition:
+        return "insert partition"
```

Comment:
```suggestion
        return "insert partitions"
```
don't want to confuse people that we only insert a single partition